### PR TITLE
Add global-roles POST/PATCH/DELETE docs

### PR DESCRIPTION
**Pull request by rashidg**
+ Also fix a typo in project-roles

Refs #SDE-3021

### CODE REVIEW

File: source/includes/global_roles.md [hunk 1]
```diff
@@ -77,3 +77,180 @@ This endpoint returns a specific Global Role resource as specified by the id par
...
+Authorization: Token "YOUR SDE ACCESS TOKEN"
+
+{
+    "name": "New Global Role",
+    "description": "Can create new applications and projects.",
+    "permissions": [
+        "add_project",
+        "add_application"
+    ],
+    "is_default": "True"
```
Comment (harry-kim) [inline source/includes/global_roles.md]: should just be true without quotes?
or do we accept both

File: source/includes/global_roles.md [hunk 2]
```diff
@@ -77,3 +77,180 @@ This endpoint returns a specific Global Role resource as specified by the id par
...
+Accept: application/json
+Authorization: Token "YOUR SDE ACCESS TOKEN"
+
+{
+    "replacement": "UR1"
+}
+```
+
+``` http
+HTTP/1.1 204 CONTENT_NOT_FOUND
```
Comment (harry-kim) [inline source/includes/global_roles.md]: maybe we should write NO CONTENT, i know i wrote CONTENT_NOT_FOUND for project role delete but i noticed others are NO CONTENT.. maybe you can update both :)

File: source/includes/global_roles.md (hunk 1 above)
Comment (RouganStriker) [inline source/includes/global_roles.md]: Should be no quotes since it is a boolean

**Comment by RouganStriker** [comment]
+1 also after addressing Harry's comments

**Pull request merged**